lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -50,11 +53,7 @@ def to_representation(self, course_overview):
                 }
             },
             'course_image': course_overview.course_image_url,
-            'course_about': reverse(
-                'about_course',
-                kwargs={'course_id': course_id},
-                request=request,
-            ),
+            'course_about': None if not course_about_url else "http:{}".format(course_about_url),
```

Comment:
@wajeeha-khalid I do not have the background on why it was done that way. 
Regardless, I agree that the code can't assume that it is "http" instead of "https".
We would need to update the CSM clients to conditionally re-add the scheme if not present, and then update the server to keep it.